tests/.infra/centos-ci/rc/cico-rc-multiuser-happy-path-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+set -e
+set +x
+
+source tests/.infra/centos-ci/functional_tests_utils.sh
+
+function prepareCustomResourceFile() {
```

Comment:
Can we extract prepareCustomResourceFile() to functional util?

Reply:
I'm not sure about this.
I think we'll get a quite uncomfortable function, and we'll have more difficulties with its using than advantages. @SkorikSergey has the same opinion.

Reply:
Hmm.. but it's true, that this function is the same in all the scripts in `rc` folder, right?
What about crating util scirpt in `rc` folder, which would for now contain only this function. This way we would avoid duplication of the function and at the same separation from other use-cases (nobody will use it from other scripts like nightly&PRcheck)

Reply:
Unfortunately no, there is one difference. The _rc-ocp-oauth-test_ has additional parameter to patch  `openShiftoAuth`. Yes, we have a duplication, difference only one now, but are you sure that it is all. Really I don't know what is better for us.

Reply:
I see 3 totally duplicated parts like 
```
function prepareCustomResourceFile() {
  cd /tmp
  wget https://raw.githubusercontent.com/eclipse/che-operator/master/deploy/crds/org_v1_che_cr.yaml -O custom-resource.yaml
  sed -i "s@server:@server:\n    customCheProperties:\n      CHE_LIMITS_USER_WORKSPACES_RUN_COUNT: '-1'@g" /tmp/custom-resource.yaml
  sed -i "s/customCheProperties:/customCheProperties:\n      CHE_WORKSPACE_AGENT_DEV_INACTIVE__STOP__TIMEOUT__MS: '300000'/" /tmp/custom-resource.yaml
  sed -i "s@identityProviderImage: 'quay.io/eclipse/che-keycloak:nightly'@identityProviderImage: 'quay.io/eclipse/che-keycloak:$RELEASE_TAG'@g" /tmp/custom-resource.yaml
  sed -i "s@cheImage: ''@cheImage: 'quay.io/eclipse/che-server'@g" /tmp/custom-resource.yaml
  sed -i "s@cheImageTag: 'nightly'@cheImageTag: '$RELEASE_TAG'@g" /tmp/custom-resource.yaml
  sed -i "s@devfileRegistryImage: 'quay.io/eclipse/che-devfile-registry:nightly'@devfileRegistryImage: 'quay.io/eclipse/che-devfile-registry:$RELEASE_VERSION'@g" /tmp/custom-resource.yaml
  sed -i "s@pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:nightly'@pluginRegistryImage: 'quay.io/eclipse/che-plugin-registry:$RELEASE_VERSION'@g " /tmp/custom-resource.yaml
  cat /tmp/custom-resource.yaml
}
```
And one difference in the cico-rc-ocp-oauth-test.sh it is : `  sed -i "s@openShiftoAuth: false@openShiftoAuth: true@g" /tmp/custom-resource.yaml`
I don't see any uncomfortable. As quick solution we can extract prepareCustomResourceFile() and add 1 param like **set-openShiftoAuth** . With this param sed add changes for the openshift.

Reply:
OK.

Reply:
Done.